packages/components/src/InputNumber/InputNumber.test.tsx

```diff
@@ -31,41 +29,7 @@ it("renders an input type number", () => {
         onFocus={[Function]}
         onKeyDown={[Function]}
         type="number"
-        value="123"
-      />
-    </div>
-  `);
-});
-
-it("renders an error", () => {
```

Comment:
Why remove this? Just because it's testing testing a bit beyond this component's concerns?

Reply:
This was removed as it was testing the old way that errors were presented. This snapshot didn't include an error at all. 

I was going to rewrite it, however there were 2 more useful tests further down that tested the error states